src/libraries/Common/src/System/Net/Http/X509ResourceClient.cs

```diff
@@ -80,6 +81,73 @@ internal static partial class X509ResourceClient
             return null;
         }
 
+        private const string SocketsHttpHandlerTypeName = "System.Net.Http.SocketsHttpHandler, System.Net.Http";
+        private const string HttpMessageHandlerTypeName = "System.Net.Http.HttpMessageHandler, System.Net.Http";
+        private const string HttpClientTypeName = "System.Net.Http.HttpClient, System.Net.Http";
+        private const string HttpRequestMessageTypeName = "System.Net.Http.HttpRequestMessage, System.Net.Http";
+        private const string HttpResponseMessageTypeName = "System.Net.Http.HttpResponseMessage, System.Net.Http";
+        private const string HttpResponseHeadersTypeName = "System.Net.Http.Headers.HttpResponseHeaders, System.Net.Http";
+        private const string HttpContentTypeName = "System.Net.Http.HttpContent, System.Net.Http";
+        private const string TaskOfHttpResponseMessageTypeName = "System.Threading.Tasks.Task`1[[System.Net.Http.HttpResponseMessage, System.Net.Http]], System.Runtime";
+
+        [UnsafeAccessor(UnsafeAccessorKind.Constructor)]
+        [return: UnsafeAccessorType(SocketsHttpHandlerTypeName)]
+        private static extern object? CreateHttpHandler();
```

Comment:
Why is this `object?`? This is a normal constructor call, it should never return `null`.